va/caa.go

```diff
@@ -117,25 +117,13 @@ func parseResults(results []caaResult) (*CAASet, error) {
 	return nil, nil
 }
 
-// If the fqdn has a direct alias among the list of CNAMEs, return it; otherwise
-// return "". Note that RFC 6844 only asks for tree-climbing on direct aliases,
-// not recursive aliases.
-func findAlias(source string, cnames []*dns.CNAME) string {
-	for _, cname := range cnames {
-		if cname.Header().Name == source {
-			return cname.Target
-		}
-	}
-	return ""
-}
-
-func parent(fqdn string) string {
+func withParents(fqdn string) []string {
```

Comment:
I think `withParents` is sufficiently complicated that there should be a small unit test if possible.

Reply:
Done.